lua/weapons/gmod_tool/stools/wire_adv.lua

```diff
@@ -17,13 +17,15 @@ if CLIENT then
 	language.Add( "WireTool_width", "Width:" )
 	language.Add( "WireTool_material", "Material:" )
 	language.Add( "WireTool_colour", "Colour:" )
+	language.Add( "WireTool_stick", "Stick to surfaces" )
 	TOOL.Information = {
 		{ name = "left_0", stage = 0, text = "Select input (Shift: Select multiple; Alt: Select all)" },
 		{ name = "right_0", stage = 0, text = "Next" },
 		{ name = "reload_0", stage = 0, text = "Unlink (Alt: Unlink all)" },
 		{ name = "mwheel_0", stage = 0, text = "Mouse wheel: Next" },
 		{ name = "left_1", stage = 1, text = "Select entity" },
 		{ name = "right_1", stage = 1, text = "Add wirepoint" },
+		{ name = "f_0", stage = 1, text = "F: Remove wirepoint" },
```

Comment:
wouldn't it make more sense to say "undo"?

Reply:
Yes